### PR TITLE
Added section for neovide and skulpin

### PR DESCRIPTION
**Pull request by aclysma**
neovide isn't a game but I thought it was cool. I also added a section about skulpin (my crate for 2d rendering with vulkan/skia.) Full disclosure, neovide uses skulpin. But mostly I'm suggesting inclusion because it got a strong response here: https://www.reddit.com/r/rust/comments/fc1l3y/no_nonsense_neovim_client_in_rust/

------

_Part of #64 ("Newsletter 7: Coordination/Tracking")_

### CODE REVIEW

File: content/posts/newsletter-007/index.md [hunk 1]
```diff
@@ -553,6 +562,16 @@ For more details, please see the [`specs` changelog].
 [specs]: https://crates.io/crates/specs
 [`specs` changelog]: https://github.com/amethyst/specs/blob/0.16.1/CHANGELOG.md#0161-2020-02-18
 
+### [Skulpin][skulpin]
```
Comment (ozkriff) [inline content/posts/newsletter-007/index.md]: Skulpin was already featured in 4th newsletter: https://rust-gamedev.github.io/posts/newsletter-004/#skulpin

Are there any more recent updates?

Comment (aclysma) [reply content/posts/newsletter-007/index.md]: Just SDL2 support and support for the latest version of winit, we can leave it out

Comment (ozkriff) [reply content/posts/newsletter-007/index.md]: (sorry! 🙇)

**Comment by phaazon** [comment]
`skulpin` sounds dope! However, I don’t think `neovide` belongs in here. It should be presented in the TWiR.

**Comment by ozkriff** [comment]
Personally, I'm ok with including skulpin in the newsletter, but I'd move it from "Game Updates" category into "Library & Tooling Updates".

**Comment by ozkriff** [comment]
:timer_clock: So, if there're no objections, I'll move the neovide section to "Library & Tooling Updates" and merge the PR in a few hours.

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**